common-compose.yml

```diff
@@ -37,6 +37,7 @@ services:
       - CROMWELL_CREDENTIALS=/app/jobs/config.json
       - CAPABILITIES_CONFIG=/app/jobs/capabilities_config.json
       - CROMWELL_URL=${CROMWELL_URL}
+      - GUNICORN_CMD_ARGS=${GUNICORN_CMD_ARGS:-"--workers=5"}
```

Comment:
How are the gunicorn workers currently configured for the shim layer for cromwell and dsub?

Reply:
We don't have any special settings for the gunicorn, which means currently it is using the default value: 1 synchronous worker. This might not be sufficient in some cases. Besides, parameterizing them here can improve the deployment, since different users may have different requirements.

Reply:
I like the idea of making this a configurable option! How does this parameter get passed to gunicorn?

Reply:
Thanks! According to the documentation(http://docs.gunicorn.org/en/stable/settings.html#settings), gunicorn can read the parameters from the environment variable called `GUNICORN_CMD_ARGS `

Reply:
Oh, okay! Thanks for the explanation here